content/apps/introduction.md

```diff
@@ -0,0 +1,21 @@
+# What's an App ?
+
+::: warning
+The Apps are only available in the **6.0 version** and higher and for **SaaS** customers.
+:::
+
+Before you start developing an App, it's important to understand what Apps can do and how they interact with the PIM.
+
+An App is a service that can be easily connected to a PIM, directly from the PIM, 
```

Comment:
I think we could say a word about the authentication process here, maybe just by mentioning the secure part of it : 

```suggestion
An App is a service that can be easily and securely connected to a PIM, directly from the PIM, 
```

Reply:
I don't want to oversell the feature, what's more "secure" when using the Apps instead of the Connections ?

Reply:
Nothing I guess. My comment was just to say that we do not mention the authentication part in this article but this has been updated in your last commit. All good for me!

Reply:
What do you mean by "authentication part" ?

Reply:
the fact that the app need an oAuth2 token to interact with the PIM